python/bindings.cpp

```diff
@@ -443,5 +446,91 @@ PYBIND11_MODULE(libsonata, m) {
 
     bindStorageClass<EdgeStorage>(m, "EdgeStorage", "EdgePopulation");
 
+    py::class_<DataFrame>(m, "DataFrame", "Something easily convertible to pandas dataframe")
+        .def_readonly("data", &DataFrame::data)
+        .def_readonly("index", &DataFrame::index);
+    py::class_<SpikeReader::Population>(m, "SpikePopulation", "A population inside a SpikeReader")
+        .def(
+            "get",
+            [](const SpikeReader::Population& self) { return self.get(); },
+            "Return all spikes")
+        .def(
+            "get",
+            [](const SpikeReader::Population& self, double tstart, double tend) {
```

Comment:
Consistency with changes on cpp API
```suggestion
            [](const SpikeReader::Population& self, double tstart, double tstop) {
```